### PR TITLE
docs: update faq.md (Arabic)

### PR DESCRIPTION
**Pull request by Ref-Bit**
Redacted spelling errors, improved translation, and included the missing links.

Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by c-ehrlich** [comment]
cc @Sboonny

File: www/src/pages/ar/faq.md [hunk 1]
```diff
@@ -41,20 +41,20 @@ dir: rtl
...
 
-## لماذا هُناك ملفات بامتداد `.js` في هذا المشروع ؟
+## لماذا هُناك ملفات بامتداد `.js` في هذا المشروع؟
 
-وِفقًا لـ [T3-Axiom #3](/en/introduction#typesafety-isnt-optional) ، نَحن نَعتبر `Type Safety` مواطناََ مِن الدرجة الأولى. لِسوء الحَظ ، لا تدعم بَعض الـ Frameworks والاضافات TypeScript مما يعني أن بعض ملفات التكوين يجب أن تكون ملفات JS وهو مُر لابُد منة.
+وِفقاً لـ[T3-Axiom #3](/en/introduction#typesafety-isnt-optional) ، نَحن نَعتبر مبدأ `Type Safety` مواطناََ مِن الدرجة الأولى. لِسوء الحَظ، لا تدعم بَعض الـFrameworks والإضافات TypeScript مما يعني أن بعض ملفات الإعدادات يجب أن تكون ملفات JS وهو أمر لابُد منه.
 
-نُحن نُحاول التأكيد على أن هذه الملفات هي `Javascript ` لأسباب خَارجة عن إرادتنا وذلك عن طريق إستخدام (cjs أو mjs)، لكن لا تزال جميع ملفات js في هذا المشروع تخضع لفحص `Types` باستخدام تعليق `ts-check@` في الأعلى.
+نُؤكد على أن هذه الملفات هي ملفات Javascript لسبب ضروري، من خلال الإعلان صراحةً عن نوع كل ملف منها (`cjs` أو `mjs`) اعتماداً على ما تدعمه المكتبة التي تستَخدِمَها، بالإضافة إلى أن جميع ملفات `js` في هذا المشروع تظل تخضع لفحص `Types` باستخدام تعليق `ts-check@` في الأعلى.
```
Comment (Sboonny) [inline www/src/pages/ar/faq.md:48]:
```suggestion
نُؤكد على أن هذه الملفات هي ملفات Javascript لسبب ضروري، من خلال الإعلان صراحةً عن نوع كل ملف منها (`cjs` أو `mjs`) اعتماداً على ما تدعمه المكتبة التي تستَخدِمَها، بالإضافة إلى أن جميع ملفات `js` في هذا المشروع تظل تخضع لفحص `Types` باستخدام تعليق `@ts-check` في الأعلى.
```

**Review by Sboonny — COMMENTED** [review]
Good job, this is more concise. I have addressed the new wording for "Create T3 App" in the proofreading PR, so this is good to go after the suggested change

File: www/src/pages/ar/faq.md [hunk 2]
```diff
@@ -41,20 +41,20 @@ dir: rtl
...
+ومع ذلك، إذا كان لابُد من تَنفيذه وترغب في رؤية مشروع مَرجعي، فلدينا [مَرجع](https://github.com/juliusmarminge/t3-i18n) يُوضح كيف يمكنك إضافة i18n إلى تَطبيق T3 باستخدام [next-i18next](https://github.com/i18next/next-i18next).
 
-## لماذا نستخدم `pages/` وليس `app/` في Next.js 13؟
+## لماذا نستخدم `/pages` وليس `/app` في Next.js 13؟
 
-كما ذكرنا مُسبقا في [T3-Axiom #2](/en/introduction#bleed-responsibly)، نحن نُحب الأشياء المتطورة لكننا نُقدر الاستقرار ، فمن الصعب تَحويل الـ router الي النظام الجديد، شاهد [not a great place to bleed](https://youtu.be/mnwUbtieOuI?t=1662).
-فـ `/app` هو مُجرد لَمحة عن المستقبل ، فهو ليس جاهزًا للإنتاج ؛ غير ان الـ (API) في مَرحلة تجريبية ومن المتوقع أن تحدث له تغييرات جذرية.
+كما ذكرنا مُسبقاً في [T3-Axiom #2](/en/introduction#bleed-responsibly)، نحن نُحب الأشياء المتطورة لكننا نُقدر الاستقرار ، فمن الصعب تَحويل الـrouter إلى النظام الجديد، شاهد [not a great place to bleed](https://youtu.be/mnwUbtieOuI?t=1662).
+فـ`/app` هو مُجرد لَمحة عن المستقبل، فهو ليس جاهزاً للبيئة الإنتاجية؛ كما أن الـAPI في مَرحلة تجريبية ومن المتوقع أن تحدث له تغييرات جذرية.
```
Comment (Sboonny) [inline www/src/pages/ar/faq.md:58]:
```suggestion
فـ`/app` هو [مُجرد لَمحة من المستقبل](https://youtu.be/rnsC-12PVlM?t=818)، فهو ليس جاهزاً للبيئة الإنتاجية؛ كما أن الـAPI في مَرحلة تجريبية ومن المتوقع أن تحدث له تغييرات جذرية.
```

**Comment by juliusmarminge** [comment]
Can we get this up to date and ready to ship?

**Comment by juliusmarminge** [comment]
Stale

**Pull request closed without merging**